HW0/P6/P6B.py

```diff
@@ -15,14 +16,30 @@ def burnTime(t):
     pool = mp.Pool(P)
 
     # Use a variety of wait times
-    ratio = []
-    wait_time = []
-
+    ratio =np.zeros(7)
```

Comment:
Your graph is different from what I got, or what other people I checked with got. It might be because you are using numpy which has other performance complexities. In general the ratio should increase as wait time increases meaning parallel becomes faster as wait time increases. This is opposite of your results.